airbyte-webapp/package.json

```diff
@@ -52,6 +52,7 @@
     "react-router-dom": "^6.3.0",
     "react-select": "^4.3.1",
     "react-table": "^7.8.0",
+    "react-tether": "^2.0.8",
```

Comment:
I wonder, if we should rather use `react-popper` over `react-tether` for our code base. It's significantly wider used (5m downloads per week vs 50k) and seems to be more actively maintained. All the headlessUI docs also have examples for popper directly in it, which might make it easier to incorporate those.

Reply:
Sounds reasonable

Reply:
Just noticed `react-popper` is in maintainance mode, they list https://floating-ui.com/docs/react-dom as the official successor, which still seems to have more tracktion than `react-tether` so maybe that should be our way to go?